### PR TITLE
ENH: Added accessor functions to layout manager and factory

### PR DESCRIPTION
**Pull request by cpinter**
- Layout manager factory can be accessed from python
- View widgets can be retrieved by their layout label
- 3D views can be retrieved by name

### CODE REVIEW

**Comment by lassoan** [comment]
LGTM!

File: Libs/MRML/Widgets/qMRMLLayoutManager.cxx [hunk 1]
```diff
@@ -960,6 +960,13 @@ qMRMLThreeDWidget* qMRMLLayoutManager::threeDWidget(int id)const
     this->mrmlViewFactory("vtkMRMLViewNode")->viewWidget(id));
 }
 
+//------------------------------------------------------------------------------
+qMRMLThreeDWidget* qMRMLLayoutManager::threeDWidget(const QString& name)const
+{
+  return qobject_cast<qMRMLThreeDWidget*>(
+    this->mrmlViewFactory("vtkMRMLViewNode")->viewWidget(name));
```
Comment (jcfr) [inline Libs/MRML/Widgets/qMRMLLayoutManager.cxx]: Check for null pointer

**Comment by jcfr** [comment]
:+1:

File: Libs/MRML/Widgets/qMRMLLayoutManager.cxx (hunk 1 above)
Comment (cpinter) [reply Libs/MRML/Widgets/qMRMLLayoutManager.cxx]: OK I'll add that to the other similar functions too then.

**Comment by cpinter** [comment]
Thanks for the reviews! Integrated in https://github.com/Slicer/Slicer/commit/7ffed00965a12b3dd5d97f6f1be303c4961cc8b4 with the required changes.

**Pull request closed without merging**